v22.1/dbeaver.md

```diff
@@ -30,43 +30,52 @@ To work through this tutorial, take the following steps:
-- **SSL certificate key**: Because DBeaver is a Java application, you will need to transform your key file to the `*.pk8` format using an [OpenSSL command](https://wiki.openssl.org/index.php/Command_Line_Utilities#pkcs8_.2F_pkcs5) like the one shown below.  Once you have created the file, enter its location here.  In this example, the filename is `client.root.pk8`.
-    {% include_cached copy-clipboard.html %}
-    ~~~ console
-    $ openssl pkcs8 -topk8 -inform PEM -outform DER -in client.root.key -out client.root.pk8 -nocrypt
-    ~~~
+1. Check the **Use SSL** checkbox as shown, and fill in the text areas as follows:
+    - **Root certificate**: Use the `ca.crt` file you generated for your secure cluster.
+    - **SSL certificate**: Use a client certificate generated from your cluster's root certificate.  For the root user, this will be named `client.root.crt`.  For additional security, you may want to create a new database user and client certificate just for use with DBeaver.
+    - **SSL certificate key**: Because DBeaver is a Java application, you will need to transform your key file to the `*.pk8` format using an [OpenSSL command](https://wiki.openssl.org/index.php/Command_Line_Utilities#pkcs8_.2F_pkcs5) like the one shown below.  Once you have created the file, enter its location here.  In this example, the filename is `client.root.pk8`.
+        {% include_cached copy-clipboard.html %}
+        ~~~ console
+        $ openssl pkcs8 -topk8 -inform PEM -outform DER -in client.root.key -out client.root.pk8 -nocrypt
+        ~~~
 
-Select **require** from the **SSL mode** dropdown.  There is no need to set the **SSL Factory**, you can let DBeaver use the default.
+1. Select **verify-ca** or **verify-full** from the **SSL mode** dropdown.  There is no need to set the **SSL Factory**, you can let DBeaver use the default.
 
 ## Step 3. Test the connection settings
 
-Click **Test Connection ...**.  If everything worked, you will see a **Success** dialog like the one shown below.
+1. Click **Test Connection ...**. If you need a driver, the following dialog displays:
+
+    <img src="{{ 'images/v22.1/dbeaver-06-download-driver.png' | relative_url }}" alt="DBeaver - download driver dialog" style="border:1px solid #eee;max-width:100%" />
+
+1. Click **Download**.
+
+    After the driver downloads, if the connection was successful, you will see a **Connected** dialog.
+
+    <img src="{{ 'images/v22.1/dbeaver-04-connection-success-dialog.png' | relative_url }}" alt="DBeaver - connection success dialog" style="border:1px solid #eee;max-width:100%" />
 
-<img src="{{ 'images/v2.1/dbeaver-04-connection-success-dialog.png' | relative_url }}" alt="DBeaver - connection success dialog" style="border:1px solid #eee;max-width:100%" />
+1. Click **OK** to dismiss the dialog.
 
 ## Step 4. Start using DBeaver
 
-Click **Finish** to get started using DBeaver with CockroachDB.
+Expand the **movr** database node and navigate to the **rides** table.
```

Comment:
Similarly to above, perhaps "movr" and "rides" should be in code formatting instead? I realize they can also be considered interface elements in this context.